Dockerfile

```diff
@@ -1,29 +1,52 @@
 ARG node_version=20.13-alpine3.19
 
-FROM node:${node_version} as builder
+FROM --platform=$BUILDPLATFORM node:${node_version} as build
```

Comment:
This turns off emulation of other platforms for this build stage. This allows the yarn cache to be filled at native speed.